appveyor.yml

```diff
@@ -0,0 +1,87 @@
+environment:
+  github_release_token:
+    secure: en/UnoFHA3RsSAHM5c6zb3RAMa4V01lC3pYG1YrHSeOPFu78SUA/0lOjNe3pp8RG
```

Comment:
Nope, this string is encrypted using AppVeyor's [secure variables](https://www.appveyor.com/docs/build-configuration#secure-variables) feature.